src/commands/editor/node/updatePackageLock.ts

```diff
@@ -0,0 +1,22 @@
+import { logger } from "@atomist/automation-client";
+import { LocalProject } from "@atomist/automation-client/project/local/LocalProject";
+import { Project } from "@atomist/automation-client/project/Project";
+import * as child_process from "child_process";
+
+export async function updatePackageLock(project: Project) {
+    const result = new Promise((resolve, reject) => child_process.exec("npm install",
```

Comment:
This seems like a lot of compute/network just to update the package-lock.json. Couldn't we just update the package name and version manually?

Reply:
did it have a package-lock before? 
... yeah, I guess it does
is it accurate?
... maybe

so, probably. But the way I update package-lock.json is by running npm install, so I wanted the automation to do that.

It also has the effect of applying formatting to package.json, which masks a bug in client that messed up the spacing when we changed that.

Reply:
(I fixed the bug in client but it isn't released afaik)